lua/hop/jump_target.lua

```diff
@@ -114,9 +114,15 @@ local function create_line_indirect_jump_targets(jump_ctx, locations, opts)
   for _, jump_target in pairs(line_jump_targets) do
     locations.jump_targets[#locations.jump_targets + 1] = jump_target
 
+    local score = nil
+    if opts.custom_distance_fn ~= nil then
+      score = opts.custom_distance_fn(jump_ctx.win_ctx.cursor, jump_target.cursor) + win_bias
```

Comment:
Refactor the distance method to have the same signature as `manh_dist`.

Reply:
My thinking was that if people are defining their own function, then x_bias as a separate parameter is no longer relevant ( they can just set the multiplier as they see fit; as in the example I wrote 100 * math.abs, etc. ). Maybe it would be better to eliminate the x_bias argument entirely and bake the default into manh_distance, but I was worried that would be a compatibility issue. 

Happy to do what's being requested here, just a few thoughts.

Reply:
I wanted to place manh_distance in a module that's available already, if that's not a good place please lmk. felt reasonable since users will regularly require hint for all sorts of other things